### DISCUSSION TITLE
Failing CIS Benchmark: Grafana DB role Excessive permissions

### DISCUSSION THREAD

**Discussion by ryanpodonnell1** [grafana/grafana#64935] [General]
The grafana postgres database currently leverages a single database role which is causing a [ Postgres CIS 4.2 benchmark](https://www.tenable.com/audits/items/CIS_PostgreSQL_9.6_v1.0.0_L1_OS_Linux.audit:7fb95ee856f82a4c46aaf30629dc0537) to fail for a DB compliance scan.

Would like to submit a Feature Request to have the option to have multiple DB roles to separate DDL/DML privileges.

Postgres Version 12
Grafana Version 9.3.8

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82828. Feel free to continue the discussion around this there. Thank you!
